disnake/enums.py

```diff
@@ -822,7 +822,8 @@ class AutoModEventType(Enum):
 
 class AutoModTriggerType(Enum):
     keyword = 1
-    harmful_link = 2
+    if not TYPE_CHECKING:
+        harmful_link = 2  # obsolete/deprecated
```

Comment:
why not delete it?

Reply:
avoiding breaking changes where possible - I'd rather deprecate it now and remove it in the next version, tbqh